modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -377,6 +380,7 @@ public String toParamName(String name) {
 
     @Override
     public String toModelName(String name) {
+        name = super.toModelName(name);
```

Comment:
Personally I prefer doing it in JavaCodegen `toModelName` as it would make my life easier reviewing changes. For example if someone changes the default `toModelName`, I've to go through all codegen to see if any one of them calls `super.toModelName` to involve the default one.